src/testdrive/src/action/sql.rs

```diff
@@ -122,90 +122,92 @@ pub async fn run_sql(mut cmd: SqlCommand, state: &mut State) -> Result<ControlFl
         return Err(e);
     }
 
-    match stmt {
-        Statement::AlterDefaultPrivileges { .. }
-        | Statement::AlterOwner { .. }
-        | Statement::CreateDatabase { .. }
-        | Statement::CreateIndex { .. }
-        | Statement::CreateSchema { .. }
-        | Statement::CreateSource { .. }
-        | Statement::CreateTable { .. }
-        | Statement::CreateView { .. }
-        | Statement::CreateMaterializedView { .. }
-        | Statement::DropObjects { .. }
-        | Statement::GrantPrivileges { .. }
-        | Statement::GrantRole { .. }
-        | Statement::RevokePrivileges { .. }
-        | Statement::RevokeRole { .. } => {
-            let response = Retry::default()
-                .max_duration(Duration::from_secs(3))
-                .clamp_backoff(Duration::from_millis(500))
-                .retry_async(|_| async {
-                    reqwest::get(&format!(
-                        "http://{}/api/coordinator/check",
+    if !state.benchmarking {
+        match stmt {
```

Comment:
This is a rather long method. What about extracting the body of this if-condition to a method? What do you think?

Reply:
Sure, done